src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -26,9 +26,15 @@ data class Car(val name: String) {
         }
     }
 
-    fun currentPosition(attempt: Int): Int {
+    fun position(attempt: Int): Int {
         return _history.take(attempt).sumBy {
             if (it == CarAction.MOVE) 1 else 0
         }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return _history.take(attempt).count { it == CarAction.MOVE }
```